dags/atd_knack_data_tracker_location_updater.py

```diff
@@ -42,7 +42,7 @@
 
 with DAG(
     dag_id=f"atd_knack_data_tracker_location_updater",
-    description="Assigns signal records to CSR issues in data tracker based on CSR location",
+    description="With data from AGOL, update the Signal Engineer Area field in Knack ",
```

Comment:
It is actually [a few more things](https://github.com/cityofaustin/atd-knack-services/blob/3e3a4817df90b637d99a59dfa94b965e29504323/services/config/locations.py#L18) besides engineer areas 📍

Reply:
😬 
how about "with data from AGOL, update signal location information in knack"?

Reply:
sounds great! 🙏